opal-core-ws/src/main/java/org/obiba/opal/web/project/ProjectResource.java

```diff
@@ -166,6 +168,34 @@ public OpalAnalysesDto getProjectTableAnalyses(@PathParam("table") String table)
             .map(analysis -> Dtos.asDto(analysis).build()).collect(Collectors.toList())).build();
   }
 
+  @POST
+  @Path("/_export-analysis")
```

Comment:
There is already a project ``/analyses`` resource then path should be ``/analyses/_export``